crates/compiler/collections/src/soa.rs

```diff
@@ -1,5 +1,7 @@
 use std::usize;
 
+/// DEPRECATED - use crates/soa instead!
```

Comment:
We should make an issue to track this if you don't plan on following up on this soon.